buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -462,14 +472,77 @@ private void logProcessInfo(String prefix, ProcessHandle.Info info) {
     }
 
     private void logFileContents(String description, Path from) {
-        logger.error("{} `{}`", description, this);
-        try(Stream<String> lines = Files.lines(from, StandardCharsets.UTF_8)) {
-            lines
-                .map(line -> "  " + line)
-                .forEach(logger::error);
+        final Map<String, Integer> errorsAndWarnings = new LinkedHashMap<>();
+        LinkedList<String> ring = new LinkedList<>();
+        try (LineNumberReader reader = new LineNumberReader(Files.newBufferedReader(from))) {
+            for (String line = reader.readLine(); line != null ; line = reader.readLine()) {
+                final String lineToAdd;
+                if (line.startsWith("[")) {
+                    // check to see if the previous message (possibly combined from multiple lines) was an error or
+                    // warning as we want to show all of them
+                    if (ring.isEmpty() == false) {
+                        String previousMessage = normalizeLogLine(ring.getLast());
+                        if (MESSAGES_WE_DONT_CARE_ABOUT.stream().anyMatch(match -> previousMessage.contains(match))) {
```

Comment:
How could this ever be the case if we check for this condition before adding to the collection on line 505?

Reply:
good catch!  It was meant to allow for ignoring stack traces with their message too, but it wasn't working as intended. 
Removed now.